learn/cookbooks/running_production.mdx

```diff
@@ -3,106 +3,106 @@ title: Running Meilisearch in production — Meilisearch documentation
 description: Deploy Meilisearch in a Digital Ocean droplet. Covers installation,  server configuration, and securing your instance.
 ---
 
-# Running Meilisearch in production
+# Running a self-hosted Meilisearch project in production
 
-[Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=oss&utm_source=docs&utm_medium=running-production-oss) is the recommended way to run Meilisearch in production environments. However, it is also possible to self-host Meilisearch.
+This tutorial will guide you through setting up a production-ready Debian server in a DigitalOcean droplet. It covers installing Meilisearch and setting up a secure environment.
 
-This tutorial will guide you through setting up a Debian server, installing Meilisearch, and setting up your environment.
+<Capsule intent="note">
+[Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=oss&utm_source=docs&utm_medium=running-production-oss) is the recommended way to run Meilisearch in production environments.
+</Capsule>
 
 ## Requirements
 
-+ An up-to-date server that runs Debian 10
-+ An ssh key pair to connect to that machine
+- A DigitalOcean droplet running Debian 10
```

Comment:
Can we set Debian 12 as default suggestion?

https://endoflife.date/debian

Reply:
I'm absolutely fine with that. Are you aware of any changes between 10 and 12 that might affect this article? At a quick glance I think it should be ok, but I'm not familiar enough with Debian to assess.